examples/fields.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 func main() {
-	groupService, err := service.NewNsGroupService("1.1.1.1", "xxx", "xxx", "v1", true)
+	groupService, err := service.NewNsGroupService("1.1.1.1", "xxx", "xxx", "v1", true, false)
```

Comment:
If we used the functional options pattern for this constructor, we wouldn't impact existing consumers each time a constructor option is added.  It can be disruptive for a consumer when simply updating a dependency version to have to figure out why compilation is broken and how to fix it.  If it is not reasonable to address it as part of this change, perhaps it could be taken up as a separate enhancement?